src/components/Header/HeaderLink.tsx

```diff
@@ -1,23 +1,15 @@
-import styled, { css } from 'styled-components'
+import React, { ComponentProps, PropsWithChildren } from 'react'
 
-import { useTheme } from '../../hooks/useTheme'
 import { TextLink } from '../TextLink'
 
-export const HeaderLink = styled(TextLink).attrs({ target: '_blank', suffix: null })`
-  ${() => {
-    const { color, shadow, spacingByChar } = useTheme()
-    return css`
-      padding-inline: ${spacingByChar(0.25)};
-      color: ${color.TEXT_WHITE};
-      box-shadow: unset;
-
-      &:hover {
-        color: ${color.TEXT_WHITE};
-      }
-
-      &:focus-visible {
-        ${shadow.focusIndicatorStyles}
-      }
-    `
-  }}
-`
+type Props = PropsWithChildren<ComponentProps<typeof TextLink>>
```

Comment:
`children` を明示的に追加する必要ないので不要

```suggestion
type Props = ComponentProps<typeof TextLink>
```